lib/components/button.vue

```diff
@@ -18,7 +20,7 @@
                 ];
             },
             componentType() {
-                return (this.href || this.to) ? 'b-link' : 'button';
+                return (this.href || this.to) ? bLink : 'button';
```

Comment:
It actually doesn't need the component to be registered as it uses directly its description.
It's still a good idea to add it though. I'll do that.